src/Acme.UI/Models/Athletes/PhysioAthletesViewModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Web.PublishedModels;
+
+namespace Acme.UI.Models.Athletes
+{
+    public class PhysioAthletesViewModel
+    {
+        private List<AthleteViewModel> _athletes;
+
+        private PhysioAthletesViewModel(IPublishedContent containerItem, Physiotherapist physio, int maxAthletesToDisplay)
+        {
+            _athletes = new List<AthleteViewModel>();
+            MaxAthletesToDisplay = maxAthletesToDisplay;
+            if (containerItem != null)
+            {
+                SetUpBaseUrl(containerItem.Url);
+            }
+            SetupPhysioDataData(physio);
+        }
+
+        private void SetUpBaseUrl(string url)
+        {
+            if (url != null)
```

Comment:
Can you create a string extension method called .HasValue() 
Apply a couple of checks in it including null and empty values